pkg/pwalk/pwalk.go

```diff
@@ -29,6 +33,8 @@ type WalkFunc = filepath.WalkFunc
 // - if more than one walkFn instance will return an error, only one
 // of such errors will be propagated and returned by Walk, others
 // will be silently discarded.
+//
+// Deprecated: use [github.com/opencontainers/selinux/pkg/pwalkdir.Walk]
```

Comment:
Kudos for using doc links!

Nit: missing period at EOL.

Reply:
Let me try it again locally, but I think there may have been a bug in Golang's parser; the doc links only work when surrounded by whitespace (to prevent `map[string]string` being seen as a link), but that also meant that adding a period at the end cause it not to be seen as a link 😞 

(Haven't filed an issue for it yet, but did for another one; https://github.com/golang/go/issues/56072)